lib/_http_client.js

```diff
@@ -311,11 +312,13 @@ ClientRequest.prototype._implicitHeader = function _implicitHeader() {
                     this[outHeadersKey]);
 };
 
-ClientRequest.prototype.abort = function abort() {
-  if (!this.aborted) {
-    process.nextTick(emitAbortNT.bind(this));
+ClientRequest.prototype.destroy = function destroy(error) {
```

Comment:
I know  we talked about that before, but would you mind adding a callback to destroy? It's needed for the async iteration to work properly: https://github.com/nodejs/node/blob/2b03e1d0a5ef2a52e4d7ce550b74a961c62e185a/lib/internal/streams/async_iterator.js#L117.